shimmingtoolbox/cli/mask.py

```diff
@@ -128,3 +130,146 @@ def threshold(fname_input, output, thr):
     nib.save(nii_img, output)
     click.echo(f"The filename for the output mask is: {os.path.abspath(output)}")
     return output
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Creates a SCT (SpinalCordToolbox) mask from the input file. Depending on the shape (cylinder,"
+                       " box or Gaussian), a mask is created along z direction. To generate this mask, its center"
+                       " must be specified by the user according the method."
+                       " Return an output nifti file with SCT mask.")
+@click.option('--input', 'fname_input', type=click.Path(), required=True,
+              help="(str): Input nifti file to mask. Must be 3D. Supported extensions are .nii or .nii.gz. Example: "
+                   "data.nii.gz")
+@click.option('--output', 'fname_output', type=click.Path(), default=os.path.join(os.curdir, 'mask.nii.gz'),
+              help="(str): Name of output mask. Supported extensions are .nii or .nii.gz. Example: data.nii. (default:"
+                   " (os.curdir, 'mask.nii.gz')).")
+@click.option('--size', default='20',
+              help="(str): Size of the mask in the axial plane, given in pixel (Example: 35) or in millimeter "
+                   "(Example: 35mm). If shape=gaussian, size corresponds to sigma (Example: 45). (default: 41)")
+@click.option('--shape', type=click.Choice(['cylinder', 'box', 'gaussian']), default='cylinder',
+              help="(str): Shape of the mask. (default: cylinder)")
+@click.option('--contrast', type=click.Choice(['t1', 't2', 't2s', 'dwi']), default='t2s',
+              help="(str): Type of image contrast. (default: t2s)")
+@click.option('--centerline', type=click.Choice(['svm', 'cnn', 'viewer', 'file']), default='svm',
+              help="(str): Algorithm for centerline fitting.")
+@click.option('--file-centerline', 'file_centerline', type=str,
+              help="(str):  Input centerline file (to use with flag -centerline file). "
+                   "Example: t2_centerline_manual.nii.gz")
+@click.option('--thr', type=float,
+              help="Binarization threshold (between 0 and 1) to apply to the segmentation prediction. Set to -1 for no "
+                   "binarization (i.e. soft segmentation output). The default threshold is specific to each contrast "
+                   "and wasestimated using an optimization algorithm. More details at: "
+                   "https://github.com/sct-pipeline/deepseg-threshold.")
+@click.option('--brain', type=click.IntRange(0, 1),
+              help="(str):  Indicate if the input image contains brain sections (to speed up segmentation). "
```

Comment:
```suggestion
              help="(int):  Set to 1 if the image contains the brain (or part of it), set to 0 otherwise (to speed up the segmentation). "
```